traffic_ops/app/db/seeds.sql

```diff
@@ -22,6 +22,257 @@ insert into role (id, name, description, priv_level) values (5, 'portal', 'Porta
 insert into role (id, name, description, priv_level) values (6, 'migrations', 'database migrations user - DO NOT REMOVE', 20) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (7, 'federation', 'Role for Secondary CZF', 15) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (8, 'steering', 'Role for Steering Delivery Services', 15) ON CONFLICT DO NOTHING;
+insert into role (id, name, description, priv_level) values (9, 'root', 'Role for full capabilities - super-user ', 30) ON CONFLICT DO NOTHING;
```

Comment:
Well, I'm not sure.
What we had in mind when we created the "root" role, was a super-user. A user that has unlimited capabilities in the system.
If the admin role answers this concept, it may be enough.
However, I'm not sure the name "admin" represents these unlimited capabilities well.
What do you say?